src/hash-table.lisp

```diff
@@ -113,33 +118,75 @@
             `(gethash ,g!key ,g!hash-table)    ; accessing form
             )))
 
-
-(defun remhash (key hash-table)
-  (let ((obj (caddr hash-table))
-        (hash (funcall (cadr hash-table) key)))
-    (prog1 (in hash obj)
+;;; remhash
```

Comment:
This and a few other comments aren't really providing lot of info, right?

Reply:
Yes

Reply:
sorry, forgot to be a bit more explicit :-)  I think it would be better to remove them in that case.

Reply:
Already done :) I forget to delete work tags. Sorry